### PR TITLE
provider docs: change test to example

### PR DESCRIPTION
**Pull request by katbyte**
With the upcomming chages to docs & versioned docs figured it was time

### CODE REVIEW

File: website/docs/r/storage_share_directory.html.markdown [hunk 1]
```diff
@@ -62,5 +62,5 @@ The following attributes are exported in addition to the arguments listed above:
 Directories within an Azure Storage File Share can be imported using the `resource id`, e.g.
 
 ```shell
-terraform import azurerm_storage_share_directory.test https://tomdevsa20.file.core.windows.net/share1/directory1
+terraform import azurerm_storage_share_directory.examplehttps://tomdevsa20.file.core.windows.net/share1/directory1
```
Comment (tombuildsstuff) [inline website/docs/r/storage_share_directory.html.markdown]:
```suggestion
terraform import azurerm_storage_share_directory.example https://account1.file.core.windows.net/share1/directory1
```

File: website/docs/r/subnet.html.markdown [hunk 2]
```diff
@@ -105,5 +105,5 @@ The following attributes are exported:
 Subnets can be imported using the `resource id`, e.g.
 
 ```shell
-terraform import azurerm_subnet.testSubnet /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/virtualNetworks/myvnet1/subnets/mysubnet1
+terraform import azurerm_subnet.exampleSubnet /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/virtualNetworks/myvnet1/subnets/mysubnet1
```
Comment (tombuildsstuff) [inline website/docs/r/subnet.html.markdown]:
```suggestion
terraform import azurerm_subnet.example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/virtualNetworks/myvnet1/subnets/mysubnet1
```

File: website/docs/r/traffic_manager_endpoint.html.markdown [hunk 3]
```diff
@@ -145,5 +145,5 @@ The following attributes are exported:
 Traffic Manager Endpoints can be imported using the `resource id`, e.g.
 
 ```shell
-terraform import azurerm_traffic_manager_endpoint.testEndpoints /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/trafficManagerProfiles/mytrafficmanagerprofile1/azureEndpoints/mytrafficmanagerendpoint
+terraform import azurerm_traffic_manager_endpoint.exampleEndpoints /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/trafficManagerProfiles/mytrafficmanagerprofile1/azureEndpoints/mytrafficmanagerendpoint
```
Comment (tombuildsstuff) [inline website/docs/r/traffic_manager_endpoint.html.markdown]:
```suggestion
terraform import azurerm_traffic_manager_endpoint.example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/trafficManagerProfiles/mytrafficmanagerprofile1/azureEndpoints/mytrafficmanagerendpoint
```

File: website/docs/r/traffic_manager_profile.html.markdown [hunk 4]
```diff
@@ -122,5 +122,5 @@ The Traffic Manager is created with the location `global`.
 Traffic Manager Profiles can be imported using the `resource id`, e.g.
 
 ```shell
-terraform import azurerm_traffic_manager_profile.testProfile /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/trafficManagerProfiles/mytrafficmanagerprofile1
+terraform import azurerm_traffic_manager_profile.exampleProfile /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/trafficManagerProfiles/mytrafficmanagerprofile1
```
Comment (tombuildsstuff) [inline website/docs/r/traffic_manager_profile.html.markdown]:
```suggestion
terraform import azurerm_traffic_manager_profile.example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/trafficManagerProfiles/mytrafficmanagerprofile1
```

File: website/docs/r/user_assigned_identity.markdown [hunk 5]
```diff
@@ -56,5 +56,5 @@ The following attributes are exported:
 User Assigned Identities can be imported using the `resource id`, e.g.
 
 ```shell
-terraform import azurerm_user_assigned_identity.testIdentity /subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/acceptanceTestResourceGroup1/providers/Microsoft.ManagedIdentity/userAssignedIdentities/testIdentity
+terraform import azurerm_user_assigned_identity.exampleIdentity /subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/acceptanceTestResourceGroup1/providers/Microsoft.ManagedIdentity/userAssignedIdentities/testIdentity
```
Comment (tombuildsstuff) [inline website/docs/r/user_assigned_identity.markdown]:
```suggestion
terraform import azurerm_user_assigned_identity.example /subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/acceptanceTestResourceGroup1/providers/Microsoft.ManagedIdentity/userAssignedIdentities/testIdentity
```

File: website/docs/r/virtual_network.html.markdown [hunk 6]
```diff
@@ -142,5 +142,5 @@ The `subnet` block exports:
 Virtual Networks can be imported using the `resource id`, e.g.
 
 ```shell
-terraform import azurerm_virtual_network.testNetwork /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/virtualNetworks/myvnet1
+terraform import azurerm_virtual_network.exampleNetwork /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/virtualNetworks/myvnet1
```
Comment (tombuildsstuff) [inline website/docs/r/virtual_network.html.markdown]:
```suggestion
terraform import azurerm_virtual_network.example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/virtualNetworks/myvnet1
```

File: website/docs/r/virtual_network_peering.html.markdown [hunk 7]
```diff
@@ -154,5 +154,5 @@ Virtual Network peerings cannot be created, updated or deleted concurrently.
 Virtual Network Peerings can be imported using the `resource id`, e.g.
 
 ```shell
-terraform import azurerm_virtual_network_peering.testPeering /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/virtualNetworks/myvnet1/virtualNetworkPeerings/myvnet1peering
+terraform import azurerm_virtual_network_peering.examplePeering /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/virtualNetworks/myvnet1/virtualNetworkPeerings/myvnet1peering
```
Comment (tombuildsstuff) [inline website/docs/r/virtual_network_peering.html.markdown]:
```suggestion
terraform import azurerm_virtual_network_peering.example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/virtualNetworks/myvnet1/virtualNetworkPeerings/myvnet1peering
```

File: website/docs/r/virtual_network_peering.html.markdown [hunk 8]
```diff
@@ -15,37 +15,37 @@ resources in the linked virtual network.
...
 
-resource "azurerm_virtual_network_peering" "test2" {
+resource "azurerm_virtual_network_peering" "example-2" {
   name                      = "peer2to1"
-  resource_group_name       = "${azurerm_resource_group.test.name}"
-  virtual_network_name      = "${azurerm_virtual_network.test2.name}"
-  remote_virtual_network_id = "${azurerm_virtual_network.test1.id}"
+  resource_group_name       = "${azurerm_resource_group.example.name}"
+  virtual_network_name      = "${azurerm_virtual_network.example-2.name}"
+  remote_virtual_network_id = "${azurerm_virtual_network.example-1.id}"
```
Comment (tombuildsstuff) [inline website/docs/r/virtual_network_peering.html.markdown]: it might be making these .first and .second

**Comment by katbyte** [comment]
Addressed PR comments @tombuildsstuff

File: website/docs/r/iothub_dps.html.markdown [hunk 9]
```diff
@@ -0,0 +1,86 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_iot_dps"
+sidebar_current: "docs-azurerm-resource-messaging-iot-dps-x"
+description: |-
+  Manages an IoT Device Provisioning Service.
+---
+
+# azurerm_iot_dps
```
Comment (mbfrahry) [inline website/docs/r/iothub_dps.html.markdown]: I think this file got mislabled? This looks like a copy of [iot_dps.html.markdown](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/r/iot_dps.html.markdown)

File: website/docs/r/iothub_dps_certificate.html.markdown [hunk 10]
```diff
@@ -0,0 +1,66 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_iot_dps_certificate"
+sidebar_current: "docs-azurerm-resource-messaging-iot-dps_certificate"
+description: |-
+  Manages an IoT Device Provisioning Service Certificate.
+---
+
+# azurerm_iot_dps_certificate
```
Comment (mbfrahry) [inline website/docs/r/iothub_dps_certificate.html.markdown]: I think this file got mislabled? This looks like a copy of [iot_dps_certificate.html.markdown](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/r/iot_dps_certificate.html.markdown)

File: website/docs/r/iothub_dps.html.markdown (hunk 9 above)
Comment (katbyte) [reply website/docs/r/iothub_dps.html.markdown]: whoops, i was wondering where these files ended up 🤣

File: website/docs/r/iothub_dps_certificate.html.markdown (hunk 10 above)
Comment (katbyte) [reply website/docs/r/iothub_dps_certificate.html.markdown]: whoops, i was wondering where these files ended up 🤣

**Pull request merged**